pkg/windows/windows.go

```diff
@@ -961,3 +962,8 @@ func SplitPath(filepath string) (dir string, fileName string) {
 	splitIndex := strings.LastIndexByte(filepath, '\\') + 1
 	return filepath[:splitIndex], filepath[splitIndex:]
 }
+
+// JoinPath joins path elements into a single path separating them with the Windows file separator.
+func JoinPath(elem1, elem2 string) string {
```

Comment:
What about using [filepath.Join()](https://pkg.go.dev/path/filepath#Join)?

Reply:
Using that will use a linux path separator `/` as the WMCO is compiled for linux.

Reply:
If you do have to retain this, please add a comment.